scipy/stats/tests/test_multicomp.py

```diff
@@ -250,6 +250,89 @@ def test_alternatives(self, alternative, statistic, pvalue):
             )
             assert_allclose(res.pvalue, pvalue, atol=1e-7)
 
+    # filter warning that could happen during optimization
+    @pytest.mark.parametrize(
```

Comment:
Have we resolved this? Is it deterministic now?

Reply:
Yes all good now with the RNG. I will remove the comment.